ops/ansible/roles/bfd-server/templates/bfd-server.sh.j2

```diff
@@ -130,25 +127,14 @@ stop_service_if_failing() {
       sudo iptables -D INPUT -p tcp ! -i lo --dport 7443 -j REJECT
       log "Server started properly"
       return 0
-    elif [[ $I -lt $STARTUP_TESTING_RETRIES ]]; then
+    else
       log "Server failed to start properly, backing off"
       sleep $STARTUP_TESTING_REQ_BACKOFF_TIMEOUT
```

Comment:
You could probably simplify this further by removing the `sleep` and logging that the process is retrying. Because each of the above requests is given 15 seconds to timeout (currently configured via `STARTUP_TESTING_REQ_TIMEOUT`), I don't think we need to wait any longer.

```suggestion
      log "Server failed to start properly, retrying..."
```